internal/common/commontest/request_gen.go

```diff
@@ -3,154 +3,24 @@ package commontest
 import (
 	"fmt"
 	"net/http"
-	"os"
-	"strings"
+
+	"github.com/Contrast-Security-OSS/go-test-bench/internal/common"
 )
 
 // UnsafeRequests generates a list of requests for all unsafe
 // endpoints common to all app frameworks.
 func UnsafeRequests(addr string) ([]*http.Request, error) {
 	var reqs []*http.Request
-	type generator func(addr string) ([]*http.Request, error)
-	for _, gen := range []generator{
-		unsafeCmdInjections,
-		unsafePathTraversals,
-		unsafeSQLInjections,
-		unsafeSSRF,
-		unsafeXSS,
-	} {
-		newReqs, err := gen(addr)
-		if err != nil {
-			return nil, err
-		}
-		reqs = append(reqs, newReqs...)
-	}
-	return reqs, nil
-}
-
-func unsafeCmdInjections(addr string) ([]*http.Request, error) {
-	types := []string{"query", "cookies"}
-	fns := []string{"exec.Command", "exec.CommandContext"}
-
-	reqs := make([]*http.Request, 0, len(types)*len(fns))
-	for _, fn := range fns {
-		for _, t := range types {
-			method := http.MethodGet
-			if t == "cookies" {
-				method = http.MethodPost
-			}
-			// TODO: This route is non-standard. Flipped param order for source and sink.
-			r, err := http.NewRequest(method, fmt.Sprintf("http://%s/cmdInjection/%s/%s/unsafe", addr, fn, t), nil)
-			if err != nil {
-				return nil, fmt.Errorf("failed to create request: %w", err)
-			}
-			AddUserInput(t, r, "", "hello there!; echo hack hack hack")
-			reqs = append(reqs, r)
-		}
+	rmap := common.GetRouteMap()
+	if len(rmap) == 0 {
+		return nil, fmt.Errorf("init error - no routes returned by GetRouteMap")
```

Comment:
Should we then be using `Errorf` here? I'm not sure we are providing any formatting in this err message.

Reply:
You're right, `errors.New()` could be used instead.

Reply:
Actually, I think I'll leave this rather than adding a dependency on `errors` - any objections?